pyteal/ast/block.py

```diff
@@ -14,8 +14,16 @@
 class BlockField(Enum):
```

Comment:
I guess do not include round here since available as global already right?

Reply:
That's about right, though again, that's a choice we made at AVM level. There's no `BlkRound` constant for looking at the round in a block.  It's exactly because there's a `global`, since that's the current round, while this is accessing an old block. But the code has to look that round up by specifying the round, so there's little point in then accessing it.